components/TextFields/src/Private/MDCTextInputAllCharactersCounter.m

```diff
@@ -0,0 +1,42 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCTextInputCharacterCounter.h"
+
+#import "MDCTextInput.h"
+
+@implementation MDCTextInputAllCharactersCounter
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+  self = [super init];
+  if (self) {
+  }
+  return self;
+}
+
+- (id)copyWithZone:(NSZone *)zone {
+  MDCTextInputAllCharactersCounter *copy = [[[self class] alloc] init];
+  return copy;
+}
+
+- (void)encodeWithCoder:(NSCoder *)aCoder {
```

Comment:
Remove as this is empty?  Or call super with the expectation that others may add to this class / method?